src/main/webapp/components/api/cache-creation-controller.service.js

```diff
@@ -198,13 +209,34 @@ angular.module('managementConsole.api')
         // all exclusionList elements are not native to DMR
         if (utils.isNotNullOrUndefined(configurationElement)) {
           // ISPN-6587: Exclude type from the exclusion list for EVICTION objects, as EVICTION.type exists.
-          var exclusionList = ['is-new-node'];
+          var exclusionList = ['is-new-node', 'jdbc-type', 'jdbc-original-type'];
           if (utils.isNullOrUndefined(configurationElement['EVICTION'])) {
-            exclusionList.push['type'];
+            exclusionList.push('type');
           }
           this.addNodeComposite(steps, address, configurationElement, exclusionList, false);
         }
+      };
 
+      CacheCreationControllerClient.prototype.updateJDBCStore = function(steps, address, configuration) {
+        var jdbcType = configuration['jdbc-type'];
+        if (utils.isNullOrUndefined(jdbcType) || jdbcType.length < 1 || utils.isNullOrUndefined(configuration[jdbcType])) {
+          return;
+        }
+
+        // Add step to create/update JDBC store
+        var objectKey = jdbcType.toUpperCase().replace(/-/g, '_');
+        debugger;
```

Comment:
@vblagoje good spot, that definitely should not be here. It's a bit of a workaround for webstorm's debugger.